pyproject.toml

```diff
@@ -104,10 +105,14 @@ ignore = [
 "test/**.py" = [
     "F405",  # star import
     "F403",  # unable to detect undefined names due to star import
+    "D",     # pydocstyle
 ]
 "docs/**" = [
     "E402",  # module level import not at top of file
 ]
+"docs/**.py" = [
+    "D",     # pydocstyle
+]
```

Comment:
@garrison You should be able to put this in the above `docs/**`.

Reply:
I could, but then it wouldn't enforce the check on the notebooks.  The only thing it had a problem with IIRC was the missing docstring in `linkcode_resolve`.  Perhaps I should say `docs/conf.py` explicitly to make it clear that is the only file I wish to exempt from this check.

Reply:
> Perhaps I should say `docs/conf.py` explicitly to make it clear that is the only file I wish to exempt from this check.

I tried this and got

```
lint: commands[1]> nbqa ruff docs/
docs/how-tos/how_to_generate_exact_quasi_dists_from_sampler.ipynb:cell_1:0:1: D100 Missing docstring in public module
docs/how-tos/how_to_generate_exact_sampling_coefficients.ipynb:cell_1:0:1: D100 Missing docstring in public module
docs/how-tos/how_to_specify_cut_wires.ipynb:cell_1:0:1: D100 Missing docstring in public module
docs/tutorials/01_gate_cutting_to_reduce_circuit_width.ipynb:cell_1:0:1: D100 Missing docstring in public module
docs/tutorials/02_gate_cutting_to_reduce_circuit_depth.ipynb:cell_1:0:1: D100 Missing docstring in public module
docs/tutorials/03_wire_cutting_via_move_instruction.ipynb:cell_1:0:1: D100 Missing docstring in public module
docs/tutorials/04_automatic_cut_finding.ipynb:cell_1:0:1: D100 Missing docstring in public module
Found 7 errors.
```

Okay.  So apparently it is ignoring the notebooks even though I said `**.py`.  I will fiddle with it a bit more.

Reply:
Updated in af0b93cff34ac13d549011db2fde20d03a4f782e.